Description: Do not install versioned binaries
 We don't need them.
Forwarded: not-needed
Author: Thomas Weber <tweber@debian.org>
Reviewed-by: Rafael Laboissiere <rafael@laboissiere.net>
Last-Update: 2015-04-17
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -166,7 +166,7 @@ mkoctfile.cc: mkoctfile.in.cc Makefile
 main.cc: main.in.cc Makefile
 	$(AM_V_GEN)$(do_subst_default_vals)
 
-install-exec-hook: make-version-links
+install-exec-hook: 
 
 uninstall-local: remove-version-links
 
--- a/scripts/miscellaneous/mkoctfile.m
+++ b/scripts/miscellaneous/mkoctfile.m
@@ -148,7 +148,7 @@ function [output, status] = mkoctfile (v
   ext = octave_config_info ("EXEEXT");
 
   shell_script = fullfile (bindir,
-                           sprintf ("mkoctfile-%s%s", OCTAVE_VERSION, ext));
+                           "mkoctfile");
 
   if (! exist (shell_script, "file"))
     __gripe_missing_component__ ("mkoctfile", "mkoctfile");
--- a/scripts/pkg/private/configure_make.m
+++ b/scripts/pkg/private/configure_make.m
@@ -30,10 +30,10 @@ function configure_make (desc, packdir,
     ver = version ();
     ext = octave_config_info ("EXEEXT");
     mkoctfile_program = fullfile (octave_bindir, ...
-                                  sprintf ("mkoctfile-%s%s", ver, ext));
+                                  "mkoctfile");
     octave_config_program = fullfile (octave_bindir, ...
-                                      sprintf ("octave-config-%s%s", ver, ext));
-    octave_binary = fullfile (octave_bindir, sprintf ("octave-%s%s", ver, ext));
+                                      "octave-config");
+    octave_binary = fullfile (octave_bindir, "octave");
 
     if (! exist (mkoctfile_program, "file"))
       __gripe_missing_component__ ("pkg", "mkoctfile");
--- a/src/main.in.cc
+++ b/src/main.in.cc
@@ -433,7 +433,7 @@ main (int argc, char **argv)
   std::string octave_bindir = get_octave_bindir ();
   std::string octave_archlibdir = get_octave_archlibdir ();
   std::string octave_cli
-    = octave_bindir + dir_sep_char + "octave-cli-" OCTAVE_VERSION;
+    = octave_bindir + dir_sep_char + "octave-cli";
   std::string octave_gui = octave_archlibdir + dir_sep_char + "octave-gui";
 
 #if defined (HAVE_OCTAVE_GUI)
